.github/workflows/build.yaml

```diff
@@ -110,6 +110,8 @@ jobs:
         env:
           KUBECONFIG: /home/runner/.kube/config
           STARBOARD_CLI_LOG_LEVEL: "0"
+          STARBOARD_TEST_DOCKERHUB_USERNAME: ${{ secrets.STARBOARD_TEST_DOCKERHUB_USERNAME }}
```

Comment:
For security reasons this doesn't work for workloads triggered by PRs coming from forked repositories